packages/e2e/src/rpc-spec.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ApiPromise } from '@polkadot/api'
+import { RuntimeContext } from '@polkadot-api/observable-client'
+import { describe, expect, it } from 'vitest'
+import { dev, env, observe, setupPolkadotApi, testingPairs } from './helper.js'
+import { firstValueFrom } from 'rxjs'
+
+const testApi = await setupPolkadotApi(env.acalaV15)
+
+const { alice, bob } = testingPairs()
+
+describe('transaction_v1', async () => {
+  it('sends and executes transactions', async () => {
+    const chainHead = testApi.observableClient.chainHead$()
+
+    const api = await prepareChainForTx()
+
+    const TRANSFERRED_VALUE = 100n
+    const tx = await api.tx.balances.transferKeepAlive(bob.address, TRANSFERRED_VALUE).signAsync(alice)
```

Comment:
papi currently can only build transactions on the top-level client, which needs a codegen step to generate descriptors. This is because we can generate the call data with the metadata-builders package, but the signed extensions and signature is only handled in the top-level client. @josepot is working on a [new sub-package](https://github.com/polkadot-api/polkadot-api/pull/673) that will make it possible to create them without the top-level client, but it will take a while.

So I thought that I could create the transactions and sign them with pjs without actually sending them, and then broadcast and track them with polkadot-api (which is the part that uses the new rpc interface spec)